.buildkite/hooks/pre-command

```diff
@@ -28,6 +28,12 @@ fi
 echo "Creating var/ artifact directory..."
 docker run --rm -v /tmp/artifacts:/artifact-mount alpine:latest /bin/sh -c 'chown -R 2000 /artifact-mount/' || true
 
+if [[ "${OSTYPE}" == msys ]]; then
+    # Disable Windows Defender real-time monitoring. This is neccessary to avoid
+    # docker build significant slow down
+    powershell -Command "Set-MpPreference -DisableRealtimeMonitoring 1"
+fi
```

Comment:
this should go into the AMI building. we should not alter system configs in the repo code.